scipy/optimize/tests/test_optimize.py

```diff
@@ -1540,6 +1540,18 @@ def test_brent(self):
         x = optimize.brent(self.fun, brack=(-15, -1, 15))
         assert_allclose(x, self.solution, atol=1e-6)
 
+        with pytest.raises(ValueError, 
+            match=  "Bracketing values (xa, xb, xc) do not fulfill"
```

Comment:
Equally here, my suggestion was the recommended way (consistent with what we usually do) as more readable. On the first line you have the string and on the second line the `with` statement.